From 4eeed7de3fa90e27658c48cb06f034da43b4f057 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Tue, 4 Mar 2014 00:10:37 -0500
Subject: [PATCH] configure: use AC_PATH_TOOL to find libgcrypt-config

Since libgcrypt-config is used to find host system libraries, we should
be using AC_PATH_TOOL, not AC_PATH_PROG, for proper cross-compiling and
multilib support.

https://bugzilla.gnome.org/show_bug.cgi?id=725635
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index a861e90..8328625 100644
--- a/configure.in
+++ b/configure.in
@@ -393,7 +393,7 @@ case $host in
     CRYPTO_TESTDIR=crypto
     ;;
   *)
-    AC_PATH_PROG(LIBGCRYPT_CONFIG, libgcrypt-config, no)
+    AC_PATH_TOOL(LIBGCRYPT_CONFIG, libgcrypt-config, no)
     if test "$LIBGCRYPT_CONFIG" != "no" ; then
       LIBGCRYPT_VERSION=`$LIBGCRYPT_CONFIG --version`
       if test VERSION_TO_NUMBER(echo $LIBGCRYPT_VERSION) -lt VERSION_TO_NUMBER(echo "1.1.42")
-- 
1.9.0

